server/service_test.go

```diff
@@ -182,18 +182,18 @@ func TestWebserverMaxHeaderSize(t *testing.T) {
 }
 
 // Example good registerValidator payload
-var payloadRegisterValidator = types.SignedValidatorRegistration{
-	Message: &types.RegisterValidatorRequestMessage{
-		FeeRecipient: _HexToAddress("0xdb65fEd33dc262Fe09D9a2Ba8F80b329BA25f941"),
-		Timestamp:    1234356,
-		GasLimit:     278234191203,
-		Pubkey: _HexToPubkey(
-			"0x8a1d7b8dd64e0aafe7ea7b6c95065c9364cf99d38470c12ee807d55f7de1529ad29ce2c422e0b65e3d5a05c02caca249"),
-	},
-	// Signed by 0x4e343a647c5a5c44d76c2c58b63f02cdf3a9a0ec40f102ebc26363b4b1b95033
-	Signature: _HexToSignature(
-		"0x8209b5391cd69f392b1f02dbc03bab61f574bb6bb54bf87b59e2a85bdc0756f7db6a71ce1b41b727a1f46ccc77b213bf0df1426177b5b29926b39956114421eaa36ec4602969f6f6370a44de44a6bce6dae2136e5fb594cce2a476354264d1ea"),
-}
+// var payloadRegisterValidator = types.SignedValidatorRegistration{
+// 	Message: &types.RegisterValidatorRequestMessage{
+// 		FeeRecipient: _HexToAddress("0xdb65fEd33dc262Fe09D9a2Ba8F80b329BA25f941"),
+// 		Timestamp:    1234356,
+// 		GasLimit:     278234191203,
+// 		Pubkey: _HexToPubkey(
+// 			"0x8a1d7b8dd64e0aafe7ea7b6c95065c9364cf99d38470c12ee807d55f7de1529ad29ce2c422e0b65e3d5a05c02caca249"),
+// 	},
+// 	// Signed by 0x4e343a647c5a5c44d76c2c58b63f02cdf3a9a0ec40f102ebc26363b4b1b95033
+// 	Signature: _HexToSignature(
+// 		"0x8209b5391cd69f392b1f02dbc03bab61f574bb6bb54bf87b59e2a85bdc0756f7db6a71ce1b41b727a1f46ccc77b213bf0df1426177b5b29926b39956114421eaa36ec4602969f6f6370a44de44a6bce6dae2136e5fb594cce2a476354264d1ea"),
+// }
```

Comment:
Why did this get commented out?

Reply:
Yeah if it's not being used we should remove it.

Reply:
done